src/GaussianProcessEmulator.jl

```diff
@@ -515,4 +588,12 @@ function svd_reverse_transform_mean_cov(μ::Array{FT, 2}, σ2::Array{FT, 2}, dec
     return transformed_μ, transformed_σ2
 end
 
+function get_standardizing_factors(data::Array{FT,2}) where {FT}
```

Comment:
This is maybe best put into `src/Utilities.jl`? There are some z-score type normalizations there too.

Reply:
Good idea, it's moved now. This function is not general anyway, users really would need to supply this.